Source/WebCore/animation/ViewTimeline.cpp

```diff
@@ -74,13 +78,56 @@ Ref<ViewTimeline> ViewTimeline::createFromCSSValue(Style::BuilderState& builderS
     return adoptRef(*new ViewTimeline(nullAtom(), axis, { startInset, endInset }));
 }
 
-// FIXME: compute offset values from options.
+static std::optional<Length> lengthForInset(std::variant<RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>> inset)
+{
+    // TODO: Figure out how to convert CSSNumericValue to Length
+    if (holds_alternative<RefPtr<CSSNumericValue>>(inset))
+        auto value = get<RefPtr<CSSNumericValue>>(inset);
+    else if (holds_alternative<RefPtr<CSSKeywordValue>>(inset))
```

Comment:
This should be just an `else` with an `ASSERT(std::holds_alternative<RefPtr<CSSKeywordValue>>(inset))` since there are only two cases to handle.

Reply:
Fixed.